### ISSUE TITLE
Fast Entry still works slow for longer lists of items

### ISSUE THREAD

**Issue by porgabi** [surveyjs/survey-creator#1096]
### Are you requesting a feature, reporting a bug or asking a question?
Reporting a bug.

### What is the current behavior?
When using the fast entry feature of certain widgets in the survey editor, pasting a long list of items into the Fast Entry box causes the page to become very slow and unresponsive. Clicking on elements in the editor is delayed while the text from the clipboard is being processed, though these clicks are applied once the loading is done. This can be seen towards the end of the repro video (from 01:07).
List of items example: https://pastebin.com/raw/Hy9nuyP4

Video of bug in action: 

https://user-images.githubusercontent.com/51411356/109994915-1200f980-7d0e-11eb-8da5-594b3840028f.mp4

### What is the expected behavior?
Page and survey editor should not slow down to this extent when pasting longer text from clipboard.

### How would you reproduce the current behavior (if this is a bug)?

1. Create a dropdown question in the survey editor.
2. Go to choices, paste above list of items into Fast Entry field.
3. Observe editor and page slowing down drastically.

### Specify your

* Browser: Firefox, Chrome
* Editor version: 1.8.23

### Other
I am aware this bug was found and supposedly fixed in another issue (https://github.com/surveyjs/survey-creator/issues/903), though it still seems to persist. This was tested on multiple different machines so it's unlikely to be hardware related.

**Comment by andrewtelnov** [comment]
@porgabi I have fixed this issue by the commit above.

Thank you,
Andrew

**Comment by porgabi** [comment]
Thanks!

**Comment by porgabi** [comment]
@andrewtelnov 

I have updated the editor to the 1.8.34 version from NPM but the issue still appears to be there. The exact same thing happens as in the original bug report video.

I checked page source to make sure the correct survey-creator script version is used and it seems to be in order:
![image](https://user-images.githubusercontent.com/51411356/110824197-1fc8f880-8293-11eb-9677-955ea41516b0.png)

Do you have any ideas as to what could be behind this?

**Comment by andrewtelnov** [comment]
@porgabi I did the following. I generate in Excel over 1000 items from  item5 to item1200, go to our [website](https://surveyjs.io/create-survey), drop a dropdown question then go to choices/Fast entry and insert these items. It took about 3-4 seconds to insert them. After that everything works as expected.
The fix was is the following. If you inserting over 20 items, then we do not add all items by one with all related notication, but we reasign question.choices property with new items.

Could you please check, do you have the same issue on our website?

Thank you,
Andrew

**Comment by porgabi** [comment]
It works properly on your demo site for me as well, the same issue does not come up. Which leads me to believe it might be something to do with our platform somehow. We use the creator in an [OrchardCore](https://github.com/OrchardCMS/OrchardCore) application and haven't experienced compatibility issues of any kind so far, so this is a bit puzzling.

**Comment by andrewtelnov** [comment]
@porgabi I beleive it uses a cached version in your case. 

Thank you,
Andrew
